dll/win32/shlwapi/propbag.cpp

```diff
@@ -580,3 +580,281 @@ SHCreatePropertyBagOnRegKey(
 
     return pRegBag->QueryInterface(riid, ppvObj);
 }
+
+/**************************************************************************
+ *  SHGetIniStringUTF7W (SHLWAPI.473)
+ */
+EXTERN_C DWORD WINAPI
+SHGetIniStringUTF7W(
+    _In_opt_z_ LPCWSTR lpAppName,
+    _In_z_ LPCWSTR lpKeyName,
+    _Out_ LPWSTR lpReturnedString,
```

Comment:
```suggestion
    _Out_z_cap_post_count_(nSize, return + 1) LPWSTR lpReturnedString,
```
(I personally find it better :thinking: )
or
```suggestion
    _Out_writes_to_(nSize, return + 1) _Post_z_ LPWSTR lpReturnedString,
```
(to retain the fact the returned string will be NULL-terminated),
or,
if you don't care about conveying the NULL-termination information:
```suggestion
    _Out_writes_to_(nSize, return + 1) LPWSTR lpReturnedString,
```